src/virus_clade_utils/util/sequence.py

```diff
@@ -1,18 +1,26 @@
 """Functions for retrieving and parsing SARS-CoV-2 virus genome data."""
 
 import json
+import lzma
 import time
 import zipfile
+from pathlib import Path
 
 import polars as pl
 import structlog
+import us
 from virus_clade_utils.util.session import check_response, get_session
 
 logger = structlog.get_logger()
 
 
 def get_covid_genome_data(released_since_date: str, base_url: str, filename: str):
-    """Download genome data package from NCBI."""
+    """
+    Download genome data package from NCBI.
```

Comment:
Currently, the target data code uses the NCBI API to download sequences. A future PR will update this function and grab the sequence fasta file directly from Nextstrain (to simplify and ensure reproducibility).